pkg/docker/containers-view.jsx

```diff
@@ -27,12 +30,12 @@ var docker = require('./docker');
 var atomic = require('./atomic');
 var util = require('./util');
 var searchImage = require("./search");
-var React = require('react');
+
 var Listing = require('cockpit-components-listing.jsx');
 var Select = require('cockpit-components-select.jsx');
 var moment = require('moment');
 
-var Dropdown = React.createClass({
+var Dropdown = createReactClass({
```

Comment:
Since this PR is about to be split into smaller once now, we can afford this kind of refactoring.
Originally I skipped that to ease review.

I hope I'm not wrong, but I expect just syntax changes with this replacement.